linkerd/app/inbound/src/lib.rs

```diff
@@ -349,20 +349,29 @@ impl Config {
             ..
         } = self.proxy.clone();
 
-        // Handles requests as they are initially received by the proxy.
-        let http_admit_request = svc::layers()
```

Comment:
nit/tioli: i kind of liked having this factored out, but nbd...

Reply:
that it would split out helped hide the fact that we had redundant failfast modules  -- one in the admit layers and another inside the buffer. It's a bit easier to reason about as one big list...